views/dashboard-status-page.twig

```diff
@@ -450,7 +450,7 @@
             }
         });
 
-        var displayMediaStatusData = {{ displayMediaStatus|raw }};
+        var displayData {% if displayStatus|raw %}= {{ displayStatus|raw }}{% endif %};
```

Comment:
Should this actually be as below?

```suggestion
        var displayMediaStatusData {% if displayMediaStatus|raw %}= {{ displayMediaStatus|raw }}{% endif %};
```

Reply:
You are correct. I think I just copy-pasted incorrectly.